internal/ent/schema/statusnamespace.go

```diff
@@ -44,7 +44,8 @@ func (StatusNamespace) Fields() []ent.Field {
 		field.String("resource_provider_id").
 			GoType(gidx.PrefixedID("")).
 			Immutable().
-			Comment("The ID for the tenant for this status namespace.").
+			NotEmpty().
+			Comment("The ID of the resource provider for this status namespace.").
```

Comment:
I think down there on L55, we need to set the `Annotation` to something similar to:
```
entx.EventsHookAdditionalSubject("resourceprovider")
```
to establish the auth relationship b/w the provider_id and the [owner](https://github.com/infratographer/resource-provider-api/blob/main/internal/ent/schema/resourceprovider.go#L51-L58) in permissions.

But... the [resource-provider-api](https://github.com/infratographer/resource-provider-api/blob/main/internal/ent/schema/resourceprovider.go#L51-L58) is still using the old events